tests/framework/installer/ceph_manifests.go

```diff
@@ -2209,6 +2218,10 @@ spec:
     securePort:
     instances: ` + strconv.Itoa(replicaCount) + `
     allNodes: false
+  healthCheck:
+    bucket:
+      enabled: true
+      interval: 60s
```

Comment:
Could we also add a check to the integration tests that will look for the health status that is a result of the bucket check? We may need a much shorter interval for that purpose as well.

Reply:
Will do.

Reply:
10s just like the mon